### DISCUSSION TITLE
Is the netlify website legit?

### DISCUSSION THREAD

**Discussion by AlmightyWings232** [xgi/houdoku#299] [Q&A]
On this GitHub page, https://houdoku.org/ is listed as the official website but if you search Houdoku on google it shows https://houdoku.netlify.app/ as the first result

**Comment by MNarath1** [comment]
After looking into it it seems that https://houdoku.netlify.app/ just redirects towards https://houdoku.org/ i checked the fingerprint and it seems to be the same website best guess is that this might be a relic from before the houdoku.org domain got bought dunno for the exact why if you are sceptical just check the websites certificate yourself and compare fingerprint hashes

**Reply by MNarath1** [reply]
I would post the results of it here but my antivirus has its own root certificate and basically takes over all websites certification (signing with its own root certificate) so unless you use the same antivirus (kaspersky) the fingerprints won't be the same
